flink-dist/src/main/flink-bin/conf/logback-yarn.xml

```diff
@@ -17,8 +17,14 @@
   -->
 
 <configuration>
-    <appender name="file" class="ch.qos.logback.core.FileAppender">
+    <appender name="file" class="ch.qos.logback.core.rolling.RollingFileAppender">
         <file>${log.file}</file>
+		<rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+			<!-- daily rollover -->
+			<fileNamePattern>logFile.%d{yyyy-MM-dd}.log</fileNamePattern>
```

Comment:
should it be `<fileNamePattern>${log.file}.%d{yyyy-MM-dd}</fileNamePattern>`?

Reply:
Yea. It should be.